### ISSUE TITLE
New release on NPM

### ISSUE THREAD

**Issue by amarant** [highsource/jsonix#189]
Hi,
could you release a new version on npm ? The current one, 2.4.1, from Jan 2016 starts to be very old.

Thanks.

**Comment by fsieduc** [comment]
+10

**Comment by kierans** [comment]
I need the fixes for #50 and #135 so releasing a new version to npm would be helpful.

**Comment by mercury83** [comment]
Hello @highsource!

Please release a new version! I'd like to use the already [documented](https://github.com/highsource/jsonix/wiki/Properties#value-property) and [merged](https://github.com/highsource/jsonix/pull/175) CDATA feature.

Thanks!

**Comment by jloewe** [comment]
@highsource Any updates? I really want to use this package in my projects!

**Comment by HugoJBello** [comment]
Same here!

**Comment by highsource** [comment]
Coming soon, please hold on for a little bit more.

**Comment by Penait1** [comment]
Yes please!

**Comment by jloewe** [comment]
NPM can also resolve Git Dependencies. So as a workaround you could use:
```json
"dependencies": {
    "jsonix": "git+https://github.com/highsource/jsonix.git"
}
```

**Comment by highsource** [comment]
Just released 3.0.0: 

https://www.npmjs.com/package/jsonix
